content/2021-06-11-make-your-life-easier.md

```diff
@@ -0,0 +1,156 @@
++++
+title = "Make your life easier"
+template = "page.html"
+date = 2021-06-11T22:00:00Z
+
+[taxonomies]
+tags = ["Tech"]
++++
+
+If you write software for long enough you begin to develop instincts about the
+relative difficulty of different problems and some of those solidify into
+rules of thumb. Like all such things they are not really _rules_ but patterns
+of thinking that you can choose to deviate from when you have good reason to
+do so. The following are some that I have come across recently. I can't claim
```

Comment:
You could trim the last two sentences down a bit to:
> These following are some of the guidelines that direct my work on a daily basis.

Reply:
Aside: I _hated_ it when I started engineering and folks would say stuff like, "It's more of an art than a science," because that was really hard for me to quantify. I like guidelines. They are, at a minimum, directional.

Reply:
That's really useful perspective. I suspect that I come at it that way because I hate when people present guidelines as generally applicable and then follow up with "well, there are exceptions...". I probably over-corrected here.

Engineering to me is about _judgement_ so if that's what art is then every skilled job is an art.